src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap_square;
+  width: ($size_square * 7) + ($gap_square * 6);
+
+  &_day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $size_square;
+    height: $size_square;
+    background-color: #eee;
+    border: 1px solid #000;
+    padding: 10px;
+    transition: 0.5s;
+    &:hover {
+      transform: translateY(-20px);
+      background-color: rgb(247, 1, 255);
```

Comment:
you should stick with one color scheme, either hex or rgb, i suggest using hex cause it's widely used